modules/ocf_desktop/manifests/xsession.pp

```diff
@@ -59,14 +59,19 @@
   }
 
   # wallpaper symlink
```

Comment:
This is now for the login screen, so I think this comment should also be updated.